server/middlewares/stats.js

```diff
@@ -31,7 +31,11 @@ const trackVisits = (req, res, next) => {
 }
 
 const trackTxSubmissionCount = mung.json((body, req, res) => {
-  if (req.originalUrl === '/api/txs/submit' && req.method === 'POST') {
+  if (
+    req.header('host') === 'https://adalite.io' &&
+    req.originalUrl === '/api/txs/submit' &&
+    req.method === 'POST'
+  ) {
```

Comment:
The purpose is not to ignore requests coming from other sources, just record them apart under the key, let's say `otherTxSubmissions` with the same subkeys (successful/unsuccessful, daily/monthly/total)